releasetasks/templates/l10n.yml.tmpl

```diff
@@ -46,6 +47,32 @@
         extra:
             task_name: "{{ buildername }}_{{ chunk }}"
         {% endif %}
+-
+    # Every l10n task requires a special task to attach all artifacts
+    taskId: "{{ stableSlugId('{}_artifacts_{}'.format(buildername, chunk)) }}"
+    reruns: 5
+    task:
+        provisionerId: "null-provisioner"
+        workerType: "buildbot"
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=36) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+            - buildbot-bridge:builder-name:{{ buildername }}
```

Comment:
in taskcluster_helper.py it looks like there is no scopes defined. I'm guessing we need scope here because the task is defined outside of the mozharness call?